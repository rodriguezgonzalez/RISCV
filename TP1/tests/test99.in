9r7p9vc3a
o799p10o110crara
vr3o543p6o03vovca
cov6v65ra
200633p251636c3127pcoc72590c45o15rc798rcv4p0aa
p20c419ccro1271r9308p2cr55219va
879339p626v02cc926o2aa
8pr0r322330r858p785o807co93o0c0108rr399440povv1r24806o440pp940r43v0va
o13r06c899pa
452o531611pooa
o8483o99c64ov1773960oa
poc28r24p4c11p4c935v84ror4cc470p766oa
o11c954p0c1cpop1v1306ora
65c67r4rpo8ro5442oo728oc9o7oa
v06p2367oa
5542v461118cvo24rrv798695v05696p9768cpcvpp0crpca
2670116prrr4co1491497ra
ov8pvvr441c30r21398vp7v6571cr696cva0a
ro4v320654p4p13cc538pc3p76p8v6o8pc8oa
049c84c0857co86375rpc19c6o650o00863c9v2o04p4op08or0v4opa
4vr6v733c21vpc8288cp81v861c37r8209r2o74v2175o926697cor9r14o4pa
33p0565773r11ccrp3936v8ro9r37v5p732o5oor7o00v451o8pp8c9vr968p392o5361ca
077cv2v7popv76r25ococpa
71265cr8co8o265o0p5c51crpcv2v06vrpco6ca
3426p469cv260984c69121rp1c26p48rp25595v9c34pa
r9ov2rvro6003ccp91p553o1raq
