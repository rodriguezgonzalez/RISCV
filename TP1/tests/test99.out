0
0
0
10
10
7
8
43
0
926
926
926
24368
23513
2
1782510
1783243
1784060
1790249
1792611
1792612
1020584
1020579
720730
799097
801289
802878
802905
803180
802878
803441
